build.ps1

```diff
@@ -488,12 +488,14 @@ function Build-CMakeProject {
             $Value += " "
           }
 
-          $ArgWithForwardSlashes = $Arg.Replace("\", "/")
-          if ($ArgWithForwardSlashes.Contains(" ")) {
-            # Quote and escape the quote so it makes it through
-            $Value += "\""$ArgWithForwardSlashes\"""
-          } else {
-            $Value += $ArgWithForwardSlashes
+          if ($Arg -ne $null) {
```

Comment:
What is inserting `$null` in here? We should rather intercept it in `TryAdd-KeyValue`

Reply:
Unknown, but `$Arg` was unbound, and thus `.Replace` would fail.

Reply:
I found it! gimme a sec

Reply:
https://github.com/compnerd/swift-build/pull/639 . I wasn't hitting it because I build for codeview